src/Detector.Cli/Program.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Microsoft.Oryx.Detector.Cli
```

Comment:
@qianz2 Just FYI...I added two projects `Detector` and `Detector.Cli` temporarily so that I can make changes at my end without waiting.